packages/components/src/components/audio/audio.ts

```diff
@@ -0,0 +1,539 @@
+          ? html`<slot name="pause-icon">
+              <sd-icon
+                name="pause"
+                library="system"
+                class=${cx('w-6 h-6', this.inverted ? 'text-primary' : 'text-white')}
+              ></sd-icon>
+            </slot>`
+          : html`<slot name="play-icon">
+              <sd-icon
+                name="start"
+                library="system"
+                class=${cx('w-6 h-6', this.inverted ? 'text-primary' : 'text-white')}
+              ></sd-icon>
+            </slot>`}
+      </button>
+
+      <div class="flex items-center justify-self-end">
+        ${this.hasSlotController.test('transcript')
+          ? html`<button
+              class=${cx(
+                'mr-6 w-6 h-6 hover:cursor-pointer hover:text-primary-500',
+                this.inverted
+                  ? 'text-white focus-visible:focus-outline-inverted'
+                  : 'text-primary focus-visible:focus-outline'
+              )}
+              @click=${this.showTranscript}
+              @keydown=${this.showTranscriptKeydown}
+              tab-index="0"
+            >
+              <sd-icon
+                class="w-6 h-6"
+                name="transcript"
+                library="system"
+                label=${this.isMuted ? this.localize.term('unmute') : this.localize.term('mute')}
+              ></sd-icon>
+            </button>`
+          : null}
+
+        <button
+          class=${cx(
+            'w-6 h-6 hover:cursor-pointer hover:text-primary-500',
+            this.inverted
+              ? 'text-white focus-visible:focus-outline-inverted'
+              : 'text-primary focus-visible:focus-outline'
+          )}
+          part="volume"
+          aria-label=${this.localize.term('mute')}
+          tabindex="0"
+          @click=${this.toggleMute}
+          @keydown=${this.toggleMuteKeydown}
+        >
+          <sd-icon class="w-6 h-6" name=${this.isMuted ? 'mute' : 'volume'} library="system"></sd-icon>
+        </button>
+      </div>
+    </div>`;
+
+    const renderTimestamps = html`<div
+      class=${cx(
+        'w-full flex justify-between',
+        this.reversedLayout ? 'mb-2' : 'mt-2',
+        this.animated && this.reversedLayout && 'absolute bottom-0 left-0 mb-2',
+        this.animated && !this.reversedLayout && 'mt-2'
+      )}
+      part="timestamps"
+    >
+      <div class=${cx('current-time text-sm', this.inverted ? 'text-primary-400' : 'text-neutral-700')}>
+        ${this.currentTime}
+      </div>
+      <div class=${cx('current-time text-sm', this.inverted ? 'text-primary-400' : 'text-neutral-700')}>
+        ${this.duration}
+      </div>
+    </div>`;
+
+    return html`
+      <div
+        class=${cx('w-full flex p-12 relative', this.reversedLayout ? 'flex-col-reverse' : 'flex-col')}
+        aria-label=${this.localize.term('audioPlayer')}
+        part="audio-player"
+      >
+        <slot name="default" @slotchange="${this.updateAudioDuration}"></slot>
```

Comment:
```suggestion
        <slot name="default" @slotchange="${this.updateAudioDuration}"></slot>
```

This should be just "<slot></slot>"! And then query e. g. for the first element inside the slot.

Reply:
Updated